source/module_hsolver/diago_dav_subspace.h

```diff
@@ -31,6 +31,8 @@ class Diago_DavSubspace : public DiagH<T, Device>
 
     static int PW_DIAG_NDIM;
 
+    static double DAV_BIG_THR;
```

Comment:
changes DAV_BIG_THR to dav_big_thr, and what's the meaning of 'big'? typically we use large, not big